Rakefile

```diff
@@ -19,6 +19,10 @@ Rake::Task[:mruby].invoke unless Dir.exist?(mruby_root)
 Dir.chdir(mruby_root)
 load "#{mruby_root}/Rakefile"
 
+load "#{mruby_root}/../mrbgem.rake"
```

Comment:
Yeah, that sounds right. @toch and I will do `File` stuff to load based off the current `Rakefile`